### ISSUE TITLE
masking for date over time

### ISSUE THREAD

**Issue by GaborTorma** [nathanreyes/v-calendar#738]
Is it possible to set masks for the date over time, like this: [https://www.screencast.com/t/X11IF9kXyNbu](url)

**Comment by nathanreyes** [comment]
Not currently. Because this section is styled, it might be a better idea to provide this customization as a slot with the provided date components (year, month, day) as slot parameters that can be styled to taste.

Would that be sufficient?

**Comment by GaborTorma** [comment]
Yes, if i can change the order of parts. The correct order In hungarian language is year, month, day.
Thanx!
